priv/repo/migrations/20240705175329_add_badge_serials.exs

```diff
@@ -0,0 +1,22 @@
+defmodule Orbit.Repo.Migrations.AddBadgeSerials do
+  use Ecto.Migration
+
+  def change do
+    create table("badge_serials") do
+      add :employee_id, references("employees", on_delete: :delete_all)
+      add :badge_serial, :string
+
+      timestamps()
+    end
+
+    create index(:badge_serials, [:badge_serial])
+
+    alter table("operator_sign_ins") do
+      modify :signed_in_employee_id, references("employees", on_delete: :delete_all),
```

Comment:
**Comment (probably not actionable):** Because we want to keep old sign-ins, it's a _teeny_ bit weird we have a `delete_all` here. Even though I get why.

Reply:
Yeah, it's sort of counterintuitive.